README.md

```diff
@@ -1,7 +1,25 @@
-# cpp-starter-project
+# Yui
 
-## Bibliography
+[![Build](https://github.com/lafette21/yui-cpp/actions/workflows/build.yml/badge.svg)](https://github.com/lafette21/yui-cpp/actions/workflows/build.yml)
 
-[CMake Conan Boilerplate](https://github.com/cpp-best-practices/cmake_conan_boilerplate_template)
+## Project structure
 
-[Project Options](https://github.com/aminya/project_options)
+```
+.
+├── 3pp
```

Comment:
`3pp` or `deps`? I personally use `deps`, many projects use that.

Reply:
I like 3pp more to be honest. gRPC uses `third_party` (https://github.com/grpc/grpc/tree/master/third_party). spdlog includes it into the `include` folder (https://github.com/gabime/spdlog/tree/v1.x/include/spdlog/fmt).

Reply:
However there is a possibility to use 2pp dependencies as well so I'm not sure.